File: modelagem/uml-poo/Aula 10/br.com.uninove.empresa/src/br/com/uninove/empresa/Funcionario.java
package br.com.uninove.empresa;

public abstract class Funcionario extends PessoaFisica {
	private String cartao;
	private double salario;
		
	public Funcionario(String nome, String rg, String cartao, double salario) {
		super(nome, rg);
		this.cartao = cartao;
		this.salario = salario;
	}
	
	public String getCartao() {
		return cartao;
	}

	public void setCartao(String cartao) {
		this.cartao = cartao;
	}
	
	public double getSalario() {
		return salario;
	}

	protected double bonificacao() {
		return this.salario * 0.1;
	}
	
	public abstract void mostrarTipoClasse();
	
}

File: 02-programacao/php/03-web/06/Correção exercicio 1/Funcionario.php
<?php
class Funcionario extends PessoaFisica {
    // Atributo
    private string $cartao;
    
    // Comportamentos
    public function getCartao() : string {
        return $this->cartao;
    }

    public function setCartao(string $cartao) {
        $this->cartao = $cartao;
    }
}
?>